pop-up.css

```diff
@@ -12,45 +12,57 @@
 }
 
 .pop-up {
-  position: fixed;
-  top: 50%;
+  position: absolute;
+  top: 44%;
```

Comment:
It looks like you've had some previous reviews, which is great! Please try to use the same pull request for requesting new reviews.
Also, as requested from [the previous reviewer](https://github.com/migcm06/miguel-portfolio/pull/16):

- [ ] Could you please take a look at the sizes of the badges and buttons? They seem a bit bigger than in the template.
- [ ] Also please check the position of your text inside your buttons.

   ### Figma design:
   ![image](https://github.com/migcm06/miguel-portfolio/assets/107952724/057b48b1-56ba-4b39-b20e-17a5044fce10)

   ### Your design:
   ![image](https://github.com/migcm06/miguel-portfolio/assets/107952724/79b7b151-56eb-4c44-82ba-069f5683e060)

Reply:
I will try to explain, the thing is that I did a merge with the previous pull request because I continued working on the form validation exercise with a branch I created from the popup branch. So, when I brought the form activity and did the merge, that merge also happened, and as a result, I cannot use the same pull request.

Reply:
I undestand your point. For your future projects, let's try not to merge your pull request with previous reviews. It would also be great if you could mention the previous pull request in the summary of this one, just to explain what happened. This way, we can keep everything organized and make sure the process goes smoothly.